Backpack-SwiftUI/MapMarker/Classes/BPKPoiMapMarker.swift

```diff
@@ -0,0 +1,97 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+
+public struct BPKPoiMapMarker: View {
+    public enum State {
+        case `default`
+        case focused
+        case disabled
+        
+        var foregroundColor: BPKColor {
+            switch self {
+            case .default:
+                return .surfaceDefaultColor
+            case .focused:
+                return .coreEcoColor
```

Comment:
Is this still a color we want to use?

Reply:
yeah, unless we change the colour, but this is the one that figma's showing, there's no special token for this